portal/eproms/views.py

```diff
@@ -96,7 +96,7 @@ def assessment_engine_view(user):
     from ..models.research_study import BASE_RS_ID, EMPRO_RS_ID, ResearchStudy
     now = datetime.utcnow()
 
-    research_study_status = patient_research_study_status(user)
+    research_study_status = patient_research_study_status(user, False)
```

Comment:
flags are a little more obvious with keyword args
```suggestion
    research_study_status = patient_research_study_status(user, ignore_QB_status=False)
```